packages/gatsby-plugin-lodash/src/gatsby-node.js

```diff
@@ -1,9 +1,31 @@
 const webpackLodashPlugin = require(`lodash-webpack-plugin`)
 
 // Add Lodash webpack plugin
-exports.modifyWebpackConfig = ({ config, stage }) => {
+exports.modifyWebpackConfig = ({ config, stage }, { disabledFeatures }) => {
   if (stage === `build-javascript`) {
-    config.plugin(`Lodash`, webpackLodashPlugin, null)
+    const features = {
+      shorthands: true,
+      cloning: true,
+      currying: true,
+      caching: true,
+      collections: true,
+      exotics: true,
+      guards: true,
+      metadata: true,
+      deburring: true,
+      unicode: true,
+      chaining: true,
+      memoizing: true,
+      coercions: true,
+      flattening: true,
+      paths: true,
+      placeholders: true,
+    }
+
```

Comment:
Rock! Alphanum sorting those features may make it easier to find-stuff/debug in the future.